src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,90 @@
+package core.basesyntax.service;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService registrationService;
+
+    @BeforeAll
+    static void setUp() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @Test
+    public void register_validUser_ok() {
+        User user = new User("validLogin", "password123", 20);
+        User registeredUser = registrationService.register(user);
+        assertEquals(user, registeredUser);
+    }
+
+    @Test
+    public void register_existingLogin_notOk() {
+        User user = new User("existingUser", "password123", 20);
+        Storage.people.add(user);
+        User newUser = new User("existingUser", "password123", 20);
+        assertThrows(RegistrationException.class, () -> {
+            registrationService.register(newUser);
+        });
```

Comment:
This test checks that a `RegistrationException` is thrown when trying to register a user with an existing login. However, you are using the `Storage.people.add()` method to add the existing user to the storage. This method is not under test, so you should not rely on it to set up your test data. Instead, you should use a mock or stub for the `StorageDao` that is used by `RegistrationServiceImpl`.